LayoutTests/http/tests/workers/service/shownotification-invalid-data.html

```diff
@@ -0,0 +1,55 @@
+<head>
+<script src="/js-test-resources/js-test.js"></script>
+<script src="/resources/notifications-test-pre.js"></script>
+<script>
+description("This tests that a notification shown by a service worker fails if it is passed an invalid data option");
+jsTestIsAsync = true;
+
+function failTheTest(msg)
+{
+    testFailed(msg);
+    finishJSTest();
+}
+
+if (window.testRunner) {
+    testRunner.waitUntilDone();
```

Comment:
This is still calling `testRunner.waitUntilDone()` instead of `jsTestIsAsync = true;`